### PR TITLE
turabian 8th edition stft widya sasana.csl

### PR DESCRIPTION
**Pull request by chrystianmalau20**
…a).csl

I changed the original "Style Turabian 8th" to the Indonesian edition. This is very meaningful because it is used at the Widya Sasana College of Philosophy-Theology (STFT), Malang. I hope this will be approved. Thanks.

### CODE REVIEW

**Comment by adam3smith** [comment]
Thanks. Please don't keep opening new pull requests -- they create a ton of notification for us maintainers and a hard to follow history on the repository. 
If you're having questions/problems, please ask.

How is the style different from Turabian 8? Are these requirements documented somewhere, ideally online?

**Comment by chrystianmalau20** [comment]
> Thanks. Please don't keep opening new pull requests -- they create a ton of notification for us maintainers and a hard to follow history on the repository. If you're having questions/problems, please ask.
> 
> How is the style different from Turabian 8? Are these requirements documented somewhere, ideally online?

My mistake. I do not understand the slightest programming language, and how to use the guide on this site.
The difference with Turabian 8 style is in the default language used before, and with a few special Indonesian abbreviations.
So far, I have had a little difficulty in writing a scientific paper without the support of the Indonesian citation system. With the approval of the quote, it will help campuses and academics. Thanks.

**Comment by adam3smith** [comment]
What type of abbreviations? We could likely add these either in the main Turabian style or in in the Indonesian locale. I'd prefer to avoid creating a separate style if it's not strictly necessary (and feel free to respond in Indonesian if that's easier, even though I'll keep writing in English)

**Comment by chrystianmalau20** [comment]
> 
There are several abbreviations that are not found in the turabian 8 citation type in Indonesian, specifically for the citation method that our campus uses, namely: penterj; Diterjemahkan oleh; Diedit oleh; dalam, etc.  Turabian citation 8th only supports english. Thus, the abbreviations used in Indonesian cannot be used.
For example In Turabian 8th (full note):

**Cititation:**

 _Teologi Perjanjian Baru 1: Allah, Manusia, Kristus (Judul Asli: New Tastement Theology),_ **trans.** Lisda Tirtapraja Gamadhi (Jakarta: BPK Gunung Mulia, 2008).
 
 **Bibliography**
Guthrie, Donal. _Teologi Perjanjian Baru 1: Allah, Manusia, Kristus (Judul Asli: New Tastement Theology)._ **Translated by** Lisda Tirtapraja Gamadhi. Jakarta: BPK Gunung Mulia, 2008.

The Turabian citation should be in Indonesian, specifically the one that has been used by the campus is like this:

**Cititation:**

 _Teologi Perjanjian Baru 1: Allah, Manusia, Kristus (Judul Asli: New Tastement Theology),_ **penterj.** Lisda Tirtapraja Gamadhi (Jakarta: BPK Gunung Mulia, 2008).
 
 **Bibliography**
Guthrie, Donal. _Teologi Perjanjian Baru 1: Allah, Manusia, Kristus (Judul Asli: New Tastement Theology)._ **Diterjemahkan oleh" Lisda Tirtapraja Gamadhi. Jakarta: BPK Gunung Mulia, 2008.

**Comment by adam3smith** [comment]
CLosing because the submitting account is deleted.
